### ISSUE TITLE
DateTime comparison for publishstartdate

### ISSUE THREAD

**Issue by schotman** [sachatrauwaen/OpenContent#42]
https://github.com/sachatrauwaen/OpenContent/blob/092cd2ba230bdcfb38e6217a76cd0b297ba0c0d5/OpenContent/Components/Handlebars/HandlebarsEngine.cs#L343

Hi Sacha,

Is it possible to do a exact compare: 

DateTime.GreaterThanOrEqual(publishstartdate, DateTime.Now)

In this way the time part will also be taken into account

**Comment by sachatrauwaen** [comment]
You mean with time consideration (not only date) ?

**Comment by schotman** [comment]
Exactly, currently, when you add a time in the editor, it has no effect
